packages/create-cloudflare/templates/hello-world-python/py/wrangler.toml

```
@@ -0,0 +1,4 @@
+name = "hello-python-worker"
+main = "src/entry.py"
+compatibility_flags = ["python_workers"]
+compatibility_date = "2024-03-20"
```

I think you might want: 

```suggestion
compatibility_date = "<TBD>"
```

By default, C3 will automatically look up the latest compat date and update this field unless a specific compat date has already been specified. Setting `2024-03-20` explicitly will have the effect of holding at this date even when newer compat dates become available.